src/constants.js

```diff
@@ -11,8 +11,10 @@ module.exports = {
   AWS_REGEX: new RegExp(
     /https?:\/\/open-learning-course-data(.*)\.s3\.amazonaws.com/g
   ),
-  INPUT_COURSE_DATE_FORMAT: "YYYY/M/D H:m:s.SSS",
-  SUPPORTED_IFRAME_EMBEDS:  {
+  // eslint-disable-next-line no-irregular-whitespace
+  IRREGULAR_WHITESPACE_REGEX: new RegExp(/(\|   )+\|/g),
```

Comment:
Is this meant to match any of the whitespace characters between two `|` characters, or do the 3 whitespace characters appear consecutively?

Reply:
They appear consecutively, although I'm just now realizing that the `+` probably doesn't really do much here.  The middle space is character code 160, aka a non breaking space, and the other two are regular spaces aka character code 32.  I think the issue has to do with table cells with only an `&nbsp;` in them being fed into turndown and the result is the ascii character code for `&nbsp;`, 160.

Reply:
So I was looking at this again, and if you're interested my rationale here was to capture any amount of `|   |`.  Here's a regexr link that demonstrates it: https://regexr.com/5oq0b